config/initializers/raven.rb

```diff
@@ -0,0 +1,5 @@
+require 'raven'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.